src/player.cpp

```diff
@@ -3830,6 +3830,8 @@ int player::impact( const int force, const tripoint &p )
     // Percentage armor penetration - armor won't help much here
     // TODO: Make cushioned items like bike helmets help more
     float armor_eff = 1.0f;
+    // Shock Absorber CBM heavily reduces damage
+    bool shock_absorbers = has_bionic ( bionic_id( "bio_shock_absorber" ) );
```

Comment:
Opening bracket should stick to the `has_bionic`. Also `shock_absorbers` could be `const`.